examples/with-nestjs/apps/web/app/page.tsx

```diff
@@ -0,0 +1,158 @@
+import { Suspense } from 'react';
+import Image from 'next/image';
+
+import { Link } from '@repo/api/links/entities/link.entity';
+
+import { Card } from '@repo/ui/card';
+import { Code } from '@repo/ui/code';
+import { Button } from '@repo/ui/button';
+
+import styles from './page.module.css';
+
+const Gradient = ({
+  conic,
+  className,
+  small,
+}: Readonly<{
+  small?: boolean;
+  conic?: boolean;
+  className?: string;
+}>) => {
+  return (
+    <span
+      className={[
+        styles.gradient,
+        conic ? styles.glowConic : undefined,
+        small ? styles.gradientSmall : styles.gradientLarge,
+        className,
+      ]
+        .filter(Boolean)
+        .join(' ')}
+    />
+  );
+};
+
+const LinksSection = async () => {
+  const links: Link[] = await (
+    await fetch('http://localhost:3000/links')
```

Comment:
If the `api` server isn't running while the `web` build is attempted, this fetch fails, failing the build.

I'm running `turbo build` and seeing everything be successful except `web#build`. I then `cd apps/api && pnpm run start` and run `turbo build --filter=web` and get a successful build.

Reply:
Let me dig for few minut

Reply:
Sorry @anthonyshew, you were right,

After I cleared my caches and retried the `build` I saw the error!
I fixed it by adding a try-catch to the request, now all's good